src/EditorFeatures/CSharpTest/ConvertTupleToStruct/ConvertTupleToStructTests.cs

```diff
@@ -1809,7 +1887,82 @@ public static implicit operator (int a, int a)(NewStruct value)
         return new NewStruct(value.a, value.a);
     }
 }";
-            await TestInRegularAndScriptAsync(text, expected, options: this.PreferImplicitTypeWithInfo(), testHost: host);
+            var test = new VerifyCS.Test
+            {
+                TestCode = text,
+                FixedCode = expected,
+                TestHost = host,
+                ExpectedDiagnostics =
+                {
+                    // /0/Test0.cs(6,25): error CS8127: Tuple element names must be unique.
+                    DiagnosticResult.CompilerError("CS8127").WithSpan(6, 25, 6, 26),
+                },
+                FixedState =
```

Comment:
Looks like the fixed state has much more diagnostics than the test state. I haven't looked at what the refactoring did yet, but it's worth giving a look to see if this high number of diagnostics indicate a bug in the refactoring.

Reply:
yup.  it's somethign we could consider changing how we emit.  but this test accurately describes how we are emitting today.  in this case, the large number is due to the fact that the tuple has identical member names.  when that becomes a struct, you end up with a ton of errors about that.

Reply:
Looked on this now. This was due to duplicate tuple element names. The generated struct had duplicated the name as well, so it had that much of diagnostics. It's probably fine given the original code is problematic.

Reply:
> yup. it's somethign we could consider changing how we emit. but this test accurately describes how we are emitting today. in this case, the large number is due to the fact that the tuple has identical member names. when that becomes a struct, you end up with a ton of errors about that.

Yeah I think it's fine. While things could be improved, it's not necessary and also completely of out scope for this PR.